src/EFTools/EntityDesign/VisualStudio/ModelWizard/gui/DatabaseObjectTreeView.cs

```diff
@@ -92,24 +92,29 @@ public DatabaseObjectTreeView()
         [SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope")]
         public void ShowStatus(string message)
         {
-            if (_statusLabel == null)
+            // dpi/scaling may have changed since the last time we showed a status
+            using (DpiAwareness.EnterDpiScope(DpiAwarenessContext.SystemAware))
             {
+                if (_statusLabel != null)
```

Comment:
Nit: I think if ShowStatus() just called HideStatus in its first line it could be simplified a bit.

Reply:
Will do